File: 12. Rekursi/E. Konversi Biner/e_konversi_biner.c
#include <stdio.h>

void biner(int N)
{
    if (N == 1)
    {
        printf("1");
    }
    else
    {
        biner(N >> 1);
        printf("%d", N % 2);
    }
}

int main()
{
    int N;
    scanf("%d", &N);
    biner(N);
    return 0;
}

File: 12. Rekursi/E. Konversi Biner/e_konversi_biner.pas
program e_konversi_biner;

procedure biner(N: longint);
begin
    if N = 1 then
        write(1)
    else begin
        biner(N shr 1);
        write(N mod 2);
    end;
end;

var
    N: longint;
begin
    readln(N);
    biner(N);
end.
